### PR TITLE
First dev piece of doc

### PR DESCRIPTION
**Pull request by Nek-**
We really need a place where we specify a strict format of query/response to avoid having differences between all ci-tron request/answers.

This PR is the first draft in this way. Please comment !

I recommand you to read it in [rich diff](https://github.com/ci-tron/ci-tron/pull/30/files?short_path=0e1550c#diff-0e1550c77f122916a363d67d0a948f30)

### CODE REVIEW

File: docs/specs/1-backend-responses-format.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+-----------------------------------------
+
+As we follow http recommandations for code status, we also follow http on
+response content type specification.
+
+So when you render a response in the context of ci-tron. It must be a json
+one.
+
+This means that you almost never have to use the `Response` class of
+Symfony but you should prefere the `JsonResponse` class of ci-tron.
```
Comment (valanz) [inline docs/specs/1-backend-responses-format.md]: You could add a link to the [`JsonResponse`](https://github.com/ci-tron/ci-tron/blob/64e6226050973139707cff1114d16795e6a8eb11/src/CiTron/Symfony/HttpFoundation/JsonResponse.php) class.

File: docs/specs/1-backend-responses-format.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+Content-Type: application/json
+Content-Length: 56
+
+{"param1": "foo","param2": 42, "param_file": "bGFsYQo="}
+```
+
+**This is *not* what we do at ci-tron !**
+
+As we follow http everywhere, we also follow the standardized way to
+transfert data: `form-urlencoded`. And if you transfert a file you can use
```
Comment (valanz) [inline docs/specs/1-backend-responses-format.md]: "transfer"

File: docs/specs/1-backend-responses-format.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
+Backend response format
+=======================
+
+Ci-tron backend must be homogeneous in their response. That's why it's time
```
Comment (valanz) [inline docs/specs/1-backend-responses-format.md]: Instead of that, what about a nice table of content ? 😸

Comment (Nek-) [inline docs/specs/1-backend-responses-format.md]: done

Comment (folliked) [inline docs/specs/1-backend-responses-format.md]: maybe "in their response**S**"

Comment (Nek-) [inline docs/specs/1-backend-responses-format.md]: Maybe not :3

**Pull request merged**